RdpGamepadPlugin/RdpGamepadPlugin.vcxproj

```diff
@@ -20,7 +20,7 @@
   </ItemGroup>
   <PropertyGroup Label="Globals">
     <ProjectGuid>{E804D24F-E3EF-425F-B986-C50B9722F81F}</ProjectGuid>
-    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
```

Comment:
Isn't 10.0 the defeault for "10.0 latest installed"? I believe this is the default value on a blank project created with VS2019. In fact, I don't have that version installed, I have 10.0.18362.0.

Reply:
Nope - at least not on VS2017. 10.0 means 10.0.  And if you don't have it, you don't compile.

Reply:
Wouldn't both these issues be solved by just installing and using VS2019?